README.md

```diff
@@ -3,9 +3,9 @@
   <h1 align="center">Sequelize</h1>
 </p>
 
-[![npm version](https://badgen.net/npm/v/sequelize)](https://www.npmjs.com/package/sequelize)
+[![npm version](https://badgen.net/npm/v/@sequelize/core)](https://www.npmjs.com/package/@sequelize/core)
 [![Build Status](https://github.com/sequelize/sequelize/workflows/CI/badge.svg)](https://github.com/sequelize/sequelize/actions?query=workflow%3ACI)
-[![npm downloads](https://badgen.net/npm/dm/sequelize)](https://www.npmjs.com/package/sequelize)
+[![npm downloads](https://badgen.net/npm/dm/@sequelize/core)](https://www.npmjs.com/package/@sequelize/core)
 [![contributors](https://img.shields.io/github/contributors/sequelize/sequelize)](https://github.com/sequelize/sequelize/graphs/contributors)
 [![Open Collective](https://img.shields.io/opencollective/backers/sequelize)](https://opencollective.com/sequelize#section-contributors)
```

Comment:
Lines 41 and 43 also need to be updated. Maybe temporarily make the distinction between how to install v6 and how to install v7 (alpha) as well, I made a suggestion in docs/manual/core-concepts/getting-started.md for this as well